load-test-framework/src/main/java/com/google/pubsub/flic/controllers/Client.java

```diff
@@ -111,6 +111,7 @@ public static String getTopicSuffix(ClientType clientType) {
       case CPS_GCLOUD_JAVA_SUBSCRIBER:
       case CPS_GCLOUD_PYTHON_PUBLISHER:
       case CPS_GCLOUD_GO_PUBLISHER:
+      case CPS_GCLOUD_GO_SUBSCRIBER:
```

Comment:
Do you mean "suffix"? Done, but I'm not sure how this changes things. I could test go end-to-end before, unless you mean run both go and java end-to-end at the same time?